fluent-resmgr/src/resource_manager.rs

```diff
@@ -74,14 +78,27 @@ impl ResourceManager {
         &self,
         locales: Vec<LanguageIdentifier>,
         resource_ids: Vec<String>,
-    ) -> FluentBundle<&FluentResource> {
+    ) -> Result<FluentBundle<&FluentResource>, Vec<ResourceManagerError>> {
+        let mut errors: Vec<ResourceManagerError> = vec![];
         let mut bundle = FluentBundle::new(locales.clone());
-        for res_id in &resource_ids {
-            println!("res_id {:?}", res_id);
-            let res = self.get_resource(res_id, &locales[0].to_string());
-            bundle.add_resource(res).unwrap();
+
+        resource_ids.iter().for_each(|res_id| {
```

Comment:
Nit: I don't think anywhere else has used the `for_each`. I think it would be simpler and more idiomatic to use a `for` loop here.